tests/validation/common.c

```diff
@@ -0,0 +1,109 @@
+/*
+ * SPDX-FileCopyrightText: Copyright (c) 2024 Intel Corporation
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+#include "mcm_dp.h"
+
+#define DEFAULT_RECV_IP "127.0.0.1"
+#define DEFAULT_RECV_PORT "9001"
+#define DEFAULT_SEND_IP "127.0.0.1"
+#define DEFAULT_SEND_PORT "9001"
+#define DEFAULT_FRAME_WIDTH 1920
+#define DEFAULT_FRAME_HEIGHT 1080
+#define DEFAULT_FPS 30.0
+#define DEFAULT_PAYLOAD_TYPE "st20"
+#define DEFAULT_LOCAL_FILE "data-sdk.264"
+#define DEFAULT_MEMIF_SOCKET_PATH "/run/mcm/mcm_rx_memif.sock"
+#define DEFAULT_MEMIF_INTERFACE_ID 0
+#define DEFAULT_PROTOCOL "auto"
+#define DEFAULT_VIDEO_FMT "yuv422p10le"
+#define DEFAULT_TOTAL_NUM 300
+#define DEFAULT_INFINITE_LOOP 0
+
+
+/* print a description of all supported options */
+void usage(FILE* fp, const char* path, int sender)
+{
+    /* take only the last portion of the path */
+    const char* basename = strrchr(path, '/');
+    basename = basename ? basename + 1 : path;
+
+    fprintf(fp, "Usage: %s [OPTION]\n", basename);
+    fprintf(fp, "to be done after the params are prepped properly");
+    fprintf(fp, "\n");
+    return;
+
+    fprintf(fp, "-w, --width=<frame_width>\t"
+                "Width of test video frame (default: %d)\n",
+        DEFAULT_FRAME_WIDTH);
+    fprintf(fp, "-h, --height=<frame_height>\t"
+                "Height of test video frame (default: %d)\n",
+        DEFAULT_FRAME_HEIGHT);
+    fprintf(fp, "-f, --fps=<video_fps>\t\t"
+                "Test video FPS (frame per second) (default: %0.2f)\n",
+        DEFAULT_FPS);
+    fprintf(fp, "-o, --protocol=protocol_type\t"
+                "Set protocol type (default: %s)\n",
+        DEFAULT_PROTOCOL);
+    fprintf(fp, "-s, --socketpath=socket_path\t"
+                "Set memif socket path (default: %s)\n",
+        DEFAULT_MEMIF_SOCKET_PATH);
+    fprintf(fp, "-d, --interfaceid=interface_id\t"
+                "Set memif conn interface id (default: %d)\n",
+        DEFAULT_MEMIF_INTERFACE_ID);
+    fprintf(fp, "-x, --pix_fmt=mcm_pix_fmt\t"
+                "Set pix_fmt conn color format (default: %s)\n",
+        DEFAULT_VIDEO_FMT);
+    fprintf(fp, "-t, --type=payload_type\t\t"
+                "Payload type (default: %s)\n",
+        DEFAULT_PAYLOAD_TYPE);
+    fprintf(fp, "-p, --port=port_number\t\t"
+                "Receive data from Port (default: %s)\n",
+        DEFAULT_RECV_PORT);
+    fprintf(fp, "\n");
+
+
+    if (sender == 1){
```

Comment:
Can you do the following instead?
```suggestion
    if (sender) {
```